webcompat/static/js/lib/wizard/step1.js

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import notify from "./notify.js";
+
+const urlField = $("#url");
+const nextStepButton = $(".next-step.step-1");
+
+//@todo remove this
+nextStepButton.removeClass("disabled");
+
+const extractPrettyUrl = url => {
+  const pathArray = $.trim(url).split("/");
+  return pathArray[2];
+};
+
+const checkUrl = () => nextStepButton.removeClass("disabled");
+const showStep = (id, data) => notify.publish("showStep", { id, data });
+
+const makeAStep = event => {
+  event.preventDefault();
+  showStep(1, { url: extractPrettyUrl(urlField.val()) });
+};
+
+urlField.on("blur input", checkUrl);
+nextStepButton.on("click", makeAStep);
+
+export default {};
```

Comment:
What's the point of exporting an empty object?

(asking because i don't know 😓 )

Reply:
This is a good point, I've importing it here, https://github.com/webcompat/webcompat.com/blob/1c9516ca8bfcaf1f93b774de56939a9ae5b02234/webcompat/static/js/lib/wizard/steps.js#L1  but not really using it since this step doesn't have any public methods (visible by default and not intended to be hidden). 

Without exporting anything, there is an error `SyntaxError: import not found: default`. 
I can probably change this and not import step1 since it's not interacting with the stepper

Reply:
Cool thanks. Since this is a WIP branch, no need to block on this right now.